linked_list/src/lib.rs

```diff
@@ -0,0 +1,210 @@
+use std::cell::RefCell;
+use std::rc::Rc;
+
+// Тип, который представляет разделяемый и изменяемый указатель на узел
+type Link<T> = Option<Rc<RefCell<Node<T>>>>;
+
+// Узел списка
```

Comment:
Док комменты пишутся вот так:

```suggestion
/// Узел списка
```

В других местах аналогично.

`//` используется обычно внутри методов. А `///` над структурами, полями и методами, чтобы показать документацию. Она еще красиво форматируется в IDE